.ostree/packages-runtime-CentOS-7.txt

```diff
@@ -0,0 +1,4 @@
+bridge-utils
```

Comment:
If these .txt files are generated by `dump_packages.py` ? maybe it is better to add a line in the commit message explain that how these txt files are  generated.

Reply:
Done.